pkgs/development/interpreters/octave/build-octave-package.nix

```diff
@@ -62,13 +62,18 @@ let
   ]
   ++ nativeBuildInputs;
 
+  passthru' = {
+    updateScript = ../../../../maintainers/scripts/update-octave-packages;
+  }
+  ++ passthru;
```

Comment:
I have never seen such a syntax for attribute sets.. Could you explain a bit what happens here? Perhaps in a Nix comment..

Reply:
`++` doesn't work on attribute sets. This is likely wrong.

Reply:
Friendly ping @KarlJoad

Reply:
@KarlJoad did you try simply `// passthru`?

Reply:
I had not tried anything. I put this down to work on some other projects. But I got something hacked together using `//` that seems to work as expected.

```sh
$ nix repl -I /home/karl/Repos
nix-repl> <nixpkgs>                                            
/home/karl/Repos/nixpkgs

nix-repl> pkgs = import <nixpkgs> {}                     

nix-repl> pkgs.octavePackages.statistics.passthru
{ updateScript = /home/karl/Repos/nixpkgs/maintainers/scripts/update-octave-packages; }
```

I am pushing two new commits that fix this issue. If we all agree, then I can rebase and squash those 2 commits into the one that introduced the `passthru.updateScript`, then do a big rebase on top of `master`, before we merge this in.